src/main/java/apoc/export/cypher/formatter/CypherFormatterUtils.java

```diff
@@ -0,0 +1,199 @@
+package apoc.export.cypher.formatter;
+
+import apoc.export.util.FormatUtils;
+import org.neo4j.graphdb.Label;
+import org.neo4j.graphdb.Node;
+import org.neo4j.helpers.collection.Iterables;
+
+import java.lang.reflect.Array;
+import java.util.*;
+
+/**
+ * @author AgileLARUS
+ *
+ * @since 16-06-2017
+ */
+public class CypherFormatterUtils {
+
+	public final static String UNIQUE_ID_LABEL = "UNIQUE IMPORT LABEL";
+	public final static String UNIQUE_ID_PROP = "UNIQUE IMPORT ID";
+	public final static String Q_UNIQUE_ID_LABEL = quote(UNIQUE_ID_LABEL);
+
+    public static class NodeInfo {
+        private Set<String> nodeIdLabels = new HashSet<>();
```

Comment:
Sets are also not ordered, and I want to keep order of properties etc. across different databases and jvm's.